pkg/cloudprovider/provider/anexia/types/types.go

```diff
@@ -17,9 +17,22 @@ limitations under the License.
 package types
 
 import (
+	"time"
+
 	providerconfigtypes "github.com/kubermatic/machine-controller/pkg/providerconfig/types"
 )
 
+const (
+	AnxTokenEnv = "ANXCLOUD_TOKEN"
+
+	CreateRequestTimeout = 15 * time.Minute
```

Comment:
Because each action type has different expected time frame, sending a request to get vm info is different than waiting till it's provisoned, normal Get request should not exceed 1 minute, VM provisioning can take up to 15 minutes. But I see that timeout 15 minutes is used to get available IP, this should be changed to 1 minute.

Reply:
Is in it actually possible to do that in a asynchronous way instead of waiting?

Reply:
It's done in asynchronus mode, as I understood and testsed the timeout is not used for the request, but for the whole operation. So this request doesn't hang for 15 minutes. However, I will double check to be sure. Thanks for pointing.